pkgs/development/python-modules/automx2/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, flask
+, flask-migrate
+, ldap3
+}:
+buildPythonPackage rec {
+  pname = "automx2";
+  version = "2022.1";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-My0YT5FLYjXSRKC0oUYNybewv6CIKr93tqMZ2fYY+5I=";
+  };
+
+  propagatedBuildInputs = [
+    flask
+    flask-migrate
+    ldap3
+  ];
+
+  meta = with lib; {
+    description = "Email client configuration made easy";
+    homepage = "https://rseichter.github.io/automx2/";
```

```suggestion
    homepage = "https://rseichter.github.io/automx2/";
    changelog = "https://github.com/rseichter/automx2/blob/${version}/CHANGELOG"
```
Please add meta.changelog for future updates.